project_dev_docs/phase1_resources/interview_script.md

```diff
@@ -0,0 +1,44 @@
+# Interview Script for User Research into Open Practices
+
+**Running time:** 30-45 mins
+
+## Context
+
+_What do we want to find out?_
+
+- Interview different people across different themes and at different levels (for example, researcher, project manager, project lead, and so on)
+- Assess the current attitudes towards open working at the Turing
+- Uncover barriers and hidden fears towards open working
+- Understand what Turing people need to address these concerns, what processes need to be put in place, for example
+- Not promoting open working, rather gaining an understanding of the barriers and drivers within the Turing community
+
+## Aims
+
+1. Understand how researchers, theme leaders, project management and research engineers at the Alan Turing Institute define open practices
+2. Understand to what extent these groups are already working in line with open practices in their daily roles
+3. Understand the main barriers and drivers for adoption of working proactices across the Alan Turing Institute
+
+## Questions
+
+1. How would you define the term "open practices" and what does it mean to you in the context of your daily work? [_Answers Aim 1_]
+2. How would you define the following terms? [_Answers Aim 1_]
+   1. Open working
+   2. Open source
+   3. Open software
+   4. Open science
+3. From your perspective, what are the key similarities between the four terms? [_Answers Aim 1_]
+4. From your perspective, what are the key differences between the four terms? [_Answers Aim 1_]
+5. What do you view as the main benefits of open working?
+6. What do you view as the main drawbacks of open working?
```

Comment:
Add some context here around "in your research". The ultimate goal is transforming research outputs into open contributions that can be improved, built upon and innovated around.